content/_data/changelogs/weekly.yml

```
@@ -14157,6 +14157,169 @@
         authors:
           - Wadeck
 
+  - version: '2.335'
+    date: 2022-02-13
+    changes:
+      - type: TODO
+        category: developer
+        pull: 6262
+        authors:
+          - dependabot[bot]
+          - basil
+        pr_title: 'Winstone 5.22: Bump Jetty from 9.4.43.v20210629 to 9.4.45.v20220203'
+        message: |-
+          Winstone 5.22: Bump Jetty from 9.4.43.v20210629 to 9.4.45.v20220203
+             Winstone 5.22 changelog: https://github.com/jenkinsci/winstone/releases/tag/winstone-5.22
+               Append to log file rather than truncating it (https://github.com/jenkinsci/winstone/pull/200)
+               Write log file and access logs in UTF-8 encoding rather than default encoding (https://github.com/jenkinsci/winstone/pull/200)
+             Jetty 9.4.45 changelog: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.45.v20220203
+      - &1
+        type: major rfe
+        category: major rfe
+        pull: 5923
+        authors:
+          - janfaracik
+        pr_title: Modernise form components
+        message: |-
+          Modernise form components.
+      - *1
+      - type: rfe
+        category: rfe
+        pull: 6254
+        issue: 67742
+        authors:
+          - NotMyFault
+        pr_title: "[JENKINS-67742] Replace computer-flash gif"
+        message: |-
+          Replace computer-flash gif icon with hourglass
+      - type: rfe
+        category: rfe
+        pull: 5901
+        authors:
+          - daniel-beck
+          - basil
+        pr_title: Adapt to jenkinsci/remoting#490
+        message: |-
+          Remove support for <code>RoleChecker#check(RoleSensitive)</code> calls added in Jenkins 2.319 again. All remoting <code>Callable</code> implementations need to perform an actual role check as documented at https://www.jenkins.io/doc/developer/security/remoting-callables/
```

```suggestion
          Remove support for <code>RoleChecker#check(RoleSensitive)</code> calls which were added again in Jenkins 2.319. All remoting <code>Callable</code> implementations need to perform an actual role check as documented at https://www.jenkins.io/doc/developer/security/remoting-callables/
```

I tried to understand the context, but I might be wrong.

---

Applied to the pull request description of https://github.com/jenkinsci/jenkins/pull/5901